parkhangs-webapp/src/features/modal/ModalMapDetail.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react'
```

Comment:
this file has been deleted in the project_5 branch and replaced by ModalParkDetail. Are you using this modal?

Reply:
nope! deleted :)